ta/Makefile

```diff
@@ -5,6 +5,8 @@ ifeq ($O,)
 $(error output path should be specified when calling this makefile)
 endif
 
+LDFLAGS=
```

Comment:
we could add `LDFLAGS ?= $(TA_LDFLAGS)` to `ta/Makefile` and `LDFLAGS ?= $(XTEST_LDFLAGS)` to `host/xtest/Makefile`. Then you can choose which flags (for which binary) you're setting.